### PR TITLE
[stable/postgresql] add comment about use existed data

### PR DESCRIPTION
**Pull request by daixiang0**
It would be helpful for users who want to use existed data with HA mode.

Signed-off-by: Xiang Dai <764524258@qq.com>

### CODE REVIEW

File: stable/postgresql/README.md [hunk 1]
```diff
@@ -275,6 +275,8 @@ The [Bitnami PostgreSQL](https://github.com/bitnami/bitnami-docker-postgresql) i
 Persistent Volume Claims are used to keep the data across deployments. This is known to work in GCE, AWS, and minikube.
 See the [Configuration](#configuration) section to configure the PVC or to disable persistence.
 
+If you already have data in it, you will failed to sync to standby nodes for all commits. If you need to use those data, please covert them to sql and import after `helm install` finished.
```
Comment (carrodher) [inline stable/postgresql/README.md]:
```suggestion
If you already have data in it, you will fail to sync to standby nodes for all commits. If you need to use those data, please covert them to sql and import after `helm install` finished.
```

File: stable/postgresql/Chart.yaml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 name: postgresql
-version: 6.3.7
-appVersion: 11.5.0
+version: 6.3.8
+appVersion: 11.5.1
```
Comment (carrodher) [inline stable/postgresql/Chart.yaml]: Why are you bumping the `appVersion`?
```suggestion
appVersion: 11.5.0
```

**Review by carrodher — CHANGES_REQUESTED** [review]

File: stable/postgresql/README.md (hunk 1 above)
Comment (javsalgar) [inline stable/postgresql/README.md]: You will failed -> you will fail.

I would give a bit more explanation to this. It is not clear the reason with just one sentence.

**Review by javsalgar — CHANGES_REQUESTED** [review]

**Comment by javsalgar** [comment]
Thanks for the PR!

File: stable/postgresql/Chart.yaml (hunk 2 above)
Comment (daixiang0) [reply stable/postgresql/Chart.yaml]: in fact i do not really know difference between appVersion and version, is it need or not?

File: stable/postgresql/README.md (hunk 1 above)
Comment (daixiang0) [reply stable/postgresql/README.md]: I have added code to explain, do any changes as you wish!

**Comment by daixiang0** [comment]
any update? @javsalgar

**Comment by javsalgar** [comment]
I see that there is no consensus regarding the fact that there's existing data.

https://github.com/bitnami/charts/issues/1414#issuecomment-533445673

 I would hold this PR until we get to the root cause.

**Comment by daixiang0** [comment]
@javsalgar hi, i open other PR to fix the root cause since those two PR cover different things.

**Comment by daixiang0** [comment]
@javsalgar please look again!

**Comment by daixiang0** [comment]
@javsalgar please look again!

**Comment by daixiang0** [comment]
@juan131 ping

**Comment by zanhsieh** [comment]
@daixiang0 
Could you bump the chart version to latest again please? Thank you.

**Comment by daixiang0** [comment]
@zanhsieh done.

**Pull request merged**